README.md

```diff
@@ -138,13 +138,26 @@ In method signatures, there should be a space after the scope (`-` or `+` symbol
 ```objc
 - (void)setExampleText:(NSString *)text image:(UIImage *)image;
 ```
+
 ## Variables
 
-Variables should be named as descriptively as possible. Single letter variable names should be avoided except as simple counter variables in `for` loops.
+Variables should be named descriptively, with the variable’s name clearly communicating what the variable _is_ and pertinent information a programmer needs to use that value properly.
+
+**For example:**
+
+* `NSString *title`: It is reasonable to assume a “title” is a string.
+* `NSString *titleHTML`: This indicates a title that may contain HTML which needs parsing for display. _“HTML” is needed for a programmer to use this variable effectively._
+* `NSAttributedString *titleAttributedString`: A title, already formatted for display. _`AttributedString` hints that this value is not just a vanilla title, and adding it could be a reasonable choice depending on context._
+* `NSDate *now`: _No further clarification is needed._
+* `NSDate *lastModifiedDate`: Simply `lastModified` can be ambiguous; depending on context, one could reasonably assume it is one of a few different types.
+* `NSURL *url` vs. `NSString *urlString`: In situations when a value can reasonably be represented by different classes, it is often useful to disambiguate in the variable’s name.
```

Comment:
> I believe `URL` and `URLString` are the NYT standard.

Offhand, I've seen a pretty equal mix across the various codebases:

![screen shot 2015-03-30 at 16 26 34](https://cloud.githubusercontent.com/assets/102904/6905977/9c788fcc-d6f9-11e4-806b-e65dc4b12e67.png)

> I'm not sure where it's declared, though. Maybe it's from one of the referenced style guides?

I can't recall seeing it specified anywhere explicitly.